<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>391</width>
    <height>610</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>391</width>
    <height>610</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>391</width>
    <height>610</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>表达式计算器</string>
  </property>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>351</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="acceptDrops">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="clearButtonEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton0">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>510</y>
     <width>81</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_sub">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>350</y>
     <width>81</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Candara Light</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton2">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>430</y>
     <width>81</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>430</y>
     <width>81</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_c">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>510</y>
     <width>81</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Candara Light</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>C</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_left">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>41</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Candara Light</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>(</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_div">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>190</y>
     <width>81</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Candara Light</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>/</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_add">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>430</y>
     <width>81</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Candara Light</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>350</y>
     <width>81</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton5">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>350</y>
     <width>81</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton7">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>81</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>7</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_delete">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>190</y>
     <width>81</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Candara Light</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&lt;-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_right">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>190</y>
     <width>41</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Candara Light</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton3">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>430</y>
     <width>81</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_equal">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>510</y>
     <width>81</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Candara Light</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>=</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton6">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>350</y>
     <width>81</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>6</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton9">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>270</y>
     <width>81</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>9</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton8">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>270</y>
     <width>81</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>8</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_mul">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>270</y>
     <width>81</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Candara Light</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>*</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_point">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>510</y>
     <width>81</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Candara Light</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>.</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_mod">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>190</y>
     <width>81</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Candara Light</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>%</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
